addons/mass_mailing/i18n/mass_mailing.pot

```diff
@@ -1434,8 +1441,6 @@ msgstr ""
 
 #. module: mass_mailing
 #: model:ir.model,name:mass_mailing.model_mailing_list
-#: model:ir.model.fields,field_description:mass_mailing.field_account_invoice_send__mailing_list_ids
-#: model:ir.model.fields,field_description:mass_mailing.field_account_tour_upload_bill__mailing_list_ids
```

Comment:
these, on the other hand, are removed because you don't have account installed while exporting. Don't remove these ones. If you want to make your life a bit easier, you export from a (community) runbot with all modules installed.